ui/packages/consul-ui/app/components/token-source/index.hbs

```diff
@@ -1,44 +1,61 @@
-<StateChart @src={{chart}} @initial={{if (eq type 'oidc') 'provider' 'secret'}} as |State Guard Action dispatch state|>
-  <Guard @name="isSecret" @cond={{action 'isSecret'}} />
+<StateChart
+  @src={{this.chart}}
+  @initial={{if (eq @type 'oidc') 'provider' 'secret'}}
+as |State Guard Action dispatch state|>
+  <Guard
+    @name="isSecret"
+    @cond={{this.isSecret}}
+  />
   {{!-- This `or` can be completely removed post 1.10 as 1.10 has optional params with default values --}}
```

Comment:
Looks like the `or` in question is gone, should the comment be removed?

Reply:
Oh! Yes 😆 , will sort before I merge, good spot!